Directory.Build.props

```diff
@@ -65,4 +65,12 @@
     <RepoTestCertificatePassword>$([System.IO.File]::ReadAllText('$(RepoTestCertificatePWD)').TrimEnd())</RepoTestCertificatePassword>
     <RepoTestCertificatePasswordRedacted>...redacted...</RepoTestCertificatePasswordRedacted>
   </PropertyGroup>
+
+  <!-- 'Clean As We Go' to prevent build agents from running out of disk space -->
+  <PropertyGroup Condition="('$(WindowsAppSDKCleanIntermediateFiles)' == '') and ('$(WindowsAppSDKBuildPipeline)' == '1')">
```

Comment:
I would remove the WindowsAppSDKBuildPipeline check (prevents local testing) and have the pipeline set WindowsAppSDKCleanIntermediateFiles